aexpect/remote.py

```diff
@@ -418,6 +418,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                             status_test_command=status_test_command,
                             client=client, host=host, port=port,
                             username=username, password=password)
+    session.output_prefix = host
```

Comment:
Hello Plamen, I can see 2 potential features in this PR:

1) logging the host
2) logging arbitrary `output_prefix` for remote executions

ad1) I agree it's essential for the logs to make sense (I'm surprised it's not doing it already), but in such case I'd suggest relying on the already stored `RemoteSession.host` argument rather than setting the `output_prefix` inside `aexpect.remote`.

ad2) might be useful to distinguish between multiple sessions, but we already have the `output_prefix` feature to do that. Also rather than creating new arguments on the fly you should create the `output_prefix` in `RemoteSession` and then let users to modify it without modifying the `session.host`.